tinygrad/engine/schedule.py

```diff
@@ -253,6 +252,7 @@ def create_schedule_with_vars(outs:List[LazyBuffer], seen:Optional[Set[LazyBuffe
       kernel_number += 1
       for out in ps.outputs: realized_lazybuffer(out, kernel_number)
     var_vals = merge_dicts([var_vals, ps.var_vals])
+    for out in ps.outputs: del out.srcs  # can only schedule once
```

Comment:
moving this because the fuzzer calls `_graph_schedule` multiple times with different ctx vars